pkgs/games/openttd/nml.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, fetchhg, python3Packages }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "nml";
+  version = "0.4.5";
+
+  src = fetchhg {
+    url = "https://hg.openttdcoop.org/nml";
```

Comment:
According to https://dev.openttdcoop.org/news/93 the repo has moved to Github. Please use `fetchFromGitHub` like in the other package.